docs/examples/fast_alps/config_on_off.toml

```diff
@@ -0,0 +1,26 @@
+[input]
+data_path = '/fefs/aswg/data/real/DL2/20200127/v0.5.1_v03/'
```

Comment:
if this analysis is intended to be run on the cluster only, providing the `data_path` and the runs numbers is redundant. The user could provide only run numbers and analysis tags to allow for an analysis of runs from different nights.

Reply:
well, most probably it will be run at the cluster most of the time, but I can't exclude other use cases. Indeed, your suggestion makes a lot of sense. Is the data location/file path guaranteed to always have pattern `/fefs/aswg/data/real/DL2/<date>/tag`?

Reply:
No, it will be changed. But it's still not clear how.

Reply:
Then I'd probably wait with the change until the data location is fixed (or there's perhaps a DB or other way to query the data location by run number and tag)

Reply:
my advice is to not try to cover all possible use cases, start with an existing one and then build from there
if today's main use case is the cluster with the existing path pattern, let's go with this - we can always update when we have more/other needs

Reply:
"temporary" and "will change soon" tend to take months if not more

Reply:
done